grails-app/views/product/_productSynonyms.gsp

```diff
@@ -91,9 +98,9 @@
                                     class="medium text"
                                     placeholder="${g.message(code: 'synonym.typeSynonym.placeholder.label', default: 'Type the synonym here')}"
                                 />
-                                <button  class="button">
+                                <button class="button">
                                     <img src="${createLinkTo(dir:'images/icons/silk', file:'add.png')}" />&nbsp;
-                                    ${warehouse.message(code:'default.button.add.label')}
+                                    <g:message code="default.button.add.label"/>
```

Comment:
is there a reason for this change? I know @jmiranda is not a big fan of changing those without a reason in terms of OBGM rebases